### PR TITLE
fixed config save bug

### PR DESCRIPTION
**Pull request by amazingalek**

### CODE REVIEW

**Review by Raicuparta — APPROVED** [review]

File: src/OWML.ModLoader/ModFinder.cs [hunk 1]
```diff
@@ -7,43 +7,43 @@
...
-		}
-	}
+        private IModData InitModData(IModManifest manifest)
+        {
+            var storage = new ModStorage(manifest, _console);
+            var config = storage.Load<ModConfig>(Constants.ModConfigFileName);
+            var defaultConfig = storage.Load<ModConfig>(Constants.ModDefaultConfigFileName) ?? new ModConfig();
+            return new ModData(manifest, config, defaultConfig, storage);
+        }
+    }
```
Comment (misternebula) [inline src/OWML.ModLoader/ModFinder.cs:48]: 😠 
should be tabs

**Comment by Raicuparta** [comment]
Fixes #337

File: src/OWML.ModLoader/ModFinder.cs (hunk 1 above)
Comment (misternebula) [reply src/OWML.ModLoader/ModFinder.cs:48]: the rest of the project is tabs

Comment (amazingalek) [reply src/OWML.ModLoader/ModFinder.cs:48]: haha :D

Comment (amazingalek) [reply src/OWML.ModLoader/ModFinder.cs]: fixed

**Pull request merged**